src/sentry/utils/distutils/commands/base.py

```diff
@@ -138,11 +138,11 @@ def _setup_npm(self):
         if node_version[2] is not None:
             log.info(u'using node ({0}) and yarn ({2})'.format(*node_version))
             self._run_command(
-                ['yarn', 'install', '--production', '--pure-lockfile']
+                ['yarn', 'install', '--pure-lockfile']
```

Comment:
I did the same thing, since I think it's nice to separate out what is used in the actual runtime vs what are build / test dependencies. But was curious for your reasoning.

Reply:
Yeah it was to split application deps from build deps.